### PR TITLE
JBPM-5427: adding specific inputs types for numeric data to prevent number format exception

### PR DESCRIPTION
**Pull request by alessioscalici**
Added new field types for numeric data types
COMMENT: merge with https://github.com/kiegroup/appformer/pull/78
@pefernan can you review it?

### CODE REVIEW

**Comment by pefernan** [comment]
ok to test

**Comment by pefernan** [comment]
+1 when tests are fixed

**Comment by jsoltes** [comment]
@alessioscalici if this also fixes the bug reported in JBPM-5427, could you please also move to corresponding state in the jira workflow (from "open" to resolved/ready for qa)?

**Comment by pefernan** [comment]
@jsoltes should we do it now or when it gets merged?

**Comment by jsoltes** [comment]
@pefernan i think it would be fine to put it in in progress once the work on the jira is started and to resolved once the pull-request is merged (there is no ready-for-qa status in the workflow).

**Comment by jsoltes** [comment]
@alessioscalici, pefernan I am also having a look at the code, there might come some comments from me, or just +1, just give me some time to look at it. :)

File: kie-wb-common-forms/kie-wb-common-forms-commons/kie-wb-common-forms-processing-engine/src/main/java/org/kie/workbench/common/forms/processing/engine/handling/impl/FormHandlerImpl.java [hunk 1]
```diff
@@ -100,7 +106,11 @@ public void registerInput( FormField formField ) {
         formFields.add( formField );
 
         if ( handlerHelper.supportsInputBinding() && formField.isBindable() ) {
-            binder.bind( widget, formField.getFieldBinding() );
+            if (valueConverter == null) {
+                binder.bind( widget, formField.getFieldBinding() );
+            } else {
+                binder.bind( widget, formField.getFieldBinding(), valueConverter );
```
Comment (jsoltes) [inline kie-wb-common-forms/kie-wb-common-forms-commons/kie-wb-common-forms-processing-engine/src/main/java/org/kie/workbench/common/forms/processing/engine/handling/impl/FormHandlerImpl.java]: this line seems to be not covered in tests

**Comment by jsoltes** [comment]
Except of the line in the else braces, which is not tested (see my comment), for me also +1...

File: kie-wb-common-forms/kie-wb-common-forms-commons/kie-wb-common-forms-processing-engine/src/main/java/org/kie/workbench/common/forms/processing/engine/handling/impl/FormHandlerImpl.java (hunk 1 above)
Comment (pefernan) [inline kie-wb-common-forms/kie-wb-common-forms-commons/kie-wb-common-forms-processing-engine/src/main/java/org/kie/workbench/common/forms/processing/engine/handling/impl/FormHandlerImpl.java]: @jsoltes can you review again?

Comment (jsoltes) [inline kie-wb-common-forms/kie-wb-common-forms-commons/kie-wb-common-forms-processing-engine/src/main/java/org/kie/workbench/common/forms/processing/engine/handling/impl/FormHandlerImpl.java]: @pefernan it is ok now

**Comment by jsoltes** [comment]
+1

**Comment by pefernan** [comment]
@hasys can you chek it please?

**Comment by pefernan** [comment]
@manstis is it ok to merge this?

**Pull request merged**